lib/src/Base/Graph/Curve.cxx

```diff
@@ -163,4 +163,24 @@ void Curve::load(Advocate & adv)
   DrawableImplementation::load(adv);
 }
 
+/* Builds a polygon wich fills the area between two curves */
+Polygon Curve::FillBetween(Sample const& dataX, Sample const& dataY1, Sample const& dataY2) {
```

Comment:
I suggest to rename the two input arguments Y1 and Y2 to clarify their meaning.
```suggestion
Polygon Curve::FillBetween(Sample const& dataX, Sample const& dataYLowerBound, Sample const& dataYLowerBound) {
```

Reply:
We keep dataY1 and dataY2 and not dataYLowerBound and dataYUpperBound because matplotlib efficiently handles polygons with intersections. y1 and y2 can therefore be alternately up and down.